moped-editor/src/views/projects/projectView/ProjectFiles.js

```diff
@@ -56,33 +51,225 @@ const useStyles = makeStyles((theme) => ({
   },
   downloadLink: {
     cursor: "pointer",
+    overflow: "hidden",
+    display: "block",
+    textOverflow: "ellipsis",
```

Comment:
i'm not sure if this is working quite right—here's a long file link in project 437
<img width="1308" alt="Screenshot 2024-10-04 at 11 48 33 AM" src="https://github.com/user-attachments/assets/776ad589-e7bc-4bbc-94ec-78c2a21cb0ce">

Reply:
ah because this isnt a link, this is just text. If it's truncated or overflow hidden, then users wouldn't be able to read the entire path